### ISSUE TITLE
Overriding hovered/keyboard selected multiselect Dropdown item label color

### ISSUE THREAD

**Issue by lehtoinen** [City-of-Helsinki/helsinki-design-system#235]
## To which feature the request relates to
The Dropdown component item label for hover/keyboard selected state is not overridable with css variables when using multiselect.

## What
Use `--menu-item-color-hover` for the hovered/keyboard selected Dropdown multiselect menu item label. Currently it uses the `--label-color` regardless of the state.

This is a problem for the multiselect variant only.

## Why
We changed the menu item hover and selected color to a darker value (for contrast). Therefore we need to change label color to white. This was possible for the mouse hover state with some selector magic (see below), but for keyboard selection there was no way that we could think of that would give us access to the label color.

```
/* Multiselect label color on hover */
.dropdown ul li:hover label {
  color: var(--asti-color-white);
}
```

## Related material
![2020-09-03 10 57 33](https://user-images.githubusercontent.com/6904260/92090073-a62dea80-edd7-11ea-9c6d-dfcf1fad928c.gif)

**Comment by ronijaakkola** [comment]
While I prefer making most of the stuff overridable, I guess this need has risen from the issue that the default HDS hover style is not accessible?

**Comment by niglu1** [comment]
This was caused by the `--menu-item-color-hover` not being passed to the Checkbox component. Fixed in #241 and will be included in the `0.13.1` release.

**Comment by lehtoinen** [comment]
> While I prefer making most of the stuff overridable, I guess this need has risen from the issue that the default HDS hover style is not accessible?

Yes, exactly the reason. I meant to point this out in the description, but failed miserably :D
